src/redux/reducers/bridge.js

```diff
@@ -144,8 +145,16 @@ const bridgeReducer = handleActions(
         toSubstrate: {
           ...state.transfers.toSubstrate,
           ...action.payload,
-        }
-      }
+        },
+      },
+    }),
+
+    [bridgeActions.getTransfersToEthereum.failure]: (state) => ({
+      ...state,
+      transfers: {
+        ...state.transfers,
+        toEthereumFailed: true,
```

Comment:
We should reset it to `false` on `.call` - in case it starts working on next try.